<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModality::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>918</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>System Monitor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::TabPosition::West</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::TabShape::Triangular</enum>
      </property>
      <property name="currentIndex">
       <number>4</number>
      </property>
      <widget class="QWidget" name="General">
       <attribute name="title">
        <string>Overview</string>
       </attribute>
       <widget class="QScrollArea" name="scrollArea">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>0</y>
          <width>641</width>
          <height>531</height>
         </rect>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOn</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>625</width>
           <height>529</height>
          </rect>
         </property>
         <widget class="QGroupBox" name="groupBox">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>611</width>
            <height>221</height>
           </rect>
          </property>
          <property name="title">
           <string>CPU</string>
          </property>
         </widget>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>220</y>
            <width>611</width>
            <height>211</height>
           </rect>
          </property>
          <property name="title">
           <string>GPU</string>
          </property>
         </widget>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>430</y>
            <width>611</width>
            <height>211</height>
           </rect>
          </property>
          <property name="title">
           <string>RAM</string>
          </property>
         </widget>
        </widget>
       </widget>
       <widget class="QComboBox" name="comboBox">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>200</y>
          <width>86</width>
          <height>26</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Default</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Darkmode</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Custom</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>180</y>
          <width>66</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>Theme:</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>260</y>
          <width>141</width>
          <height>191</height>
         </rect>
        </property>
        <property name="title">
         <string>Custom theme:</string>
        </property>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>121</width>
           <height>18</height>
          </rect>
         </property>
         <property name="text">
          <string>Background color</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>80</y>
           <width>101</width>
           <height>18</height>
          </rect>
         </property>
         <property name="text">
          <string>Font color</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>130</y>
           <width>66</width>
           <height>18</height>
          </rect>
         </property>
         <property name="text">
          <string>Font size</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>121</width>
           <height>26</height>
          </rect>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>100</y>
           <width>121</width>
           <height>26</height>
          </rect>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>150</y>
           <width>121</width>
           <height>26</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>120</width>
          <height>131</height>
         </rect>
        </property>
        <property name="title">
         <string>Hide:</string>
        </property>
        <widget class="QCheckBox" name="checkBox">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>92</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>CPU</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBox_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>92</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>GPU</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBox_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>92</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>RAM</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBox_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>80</y>
           <width>92</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>DISK</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBox_5">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>100</y>
           <width>92</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>NETWORK</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="CPU">
       <attribute name="title">
        <string>CPU</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout">
        <item row="1" column="0">
         <widget class="QLabel" name="labelCpuModel">
          <property name="text">
           <string>Model:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="labelCpuClock">
          <property name="text">
           <string>Clock Speed:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="labelCpuArch">
          <property name="text">
           <string>Architecture:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="labelCpuUsage">
          <property name="text">
           <string>CPU Usage:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="RAM">
       <attribute name="title">
        <string>RAM</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="labelRamTotal">
          <property name="text">
           <string>Total Memory:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelRamFree">
          <property name="text">
           <string>Free Memory:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="labelRamAvailable">
          <property name="text">
           <string>Available Memory:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="labelRamCached">
          <property name="text">
           <string>Cached Memory:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="GPU">
       <attribute name="title">
        <string>GPU</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="labelGpuClockSpeed">
          <property name="text">
           <string>GPU Clock Speed</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelGpuName">
          <property name="text">
           <string>GPU Name:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="labelGpuMemory">
          <property name="text">
           <string>GPU Memory:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="DISK">
       <attribute name="title">
        <string>DISK</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="labelDiskModel">
          <property name="text">
           <string>Disk Model:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelDiskSpeed">
          <property name="text">
           <string>Disk Speeds:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="NETWORK">
       <attribute name="title">
        <string>NETWORK</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout_5">
        <item row="0" column="0">
         <widget class="QLabel" name="labelNetworkSpeed">
          <property name="text">
           <string>Download:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelNetworkIP">
          <property name="text">
           <string>IPv4:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="labelNetworkIPv6">
          <property name="text">
           <string>IPv6:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="labelNetworkType">
          <property name="text">
           <string>Netowrk Type: </string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>918</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
